include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -0,0 +1,241 @@
+              typename id_type,
+              typename qual_type>
+    void read(stream_type                                                            	& stream,
+              sequence_file_input_options<seq_legal_alph_type, seq_qual_combined> const & options,
+              seq_type                                                                  & sequence,
+              id_type                                                                   & id,
+              qual_type                                                                 & SEQAN3_DOXYGEN_ONLY(qualities))
+    {
+        using stream_buff_t = std::istreambuf_iterator<typename stream_type::char_type>;
+        auto stream_view = std::ranges::subrange<decltype(stream_buff_t{stream}), decltype(stream_buff_t{})>
+                           {stream_buff_t{stream}, stream_buff_t{}};
+        auto stream_it = std::ranges::begin(stream_view);
+
+        if (!(std::ranges::equal(stream_view | view::take_until_or_throw(is_cntrl || is_blank), std::string{"LOCUS"})))
+            throw parse_error{"An entry has to start with the code word LOCUS."};
+
+        //ID
+        if constexpr (!detail::decays_to_ignore_v<id_type>)
+        {
+            if (options.embl_genbank_complete_header)
+            {
+                std::ranges::copy(std::string{"LOCUS"}, std::back_inserter(id));
+
+                while (!is_char<'O'>(*std::ranges::begin(stream_view)))
+                {
+                        std::ranges::copy(stream_view | view::take_line_or_throw
+                                                	  | view::char_to<value_type_t<id_type>>,
+                                     	                std::back_inserter(id));
+                        id.push_back('\n');
+                }
+            }
+            else
+            {
+                detail::consume(stream_view | view::take_until(!is_blank));
+                // read id
+                std::ranges::copy(stream_view | view::take_until_or_throw(is_cntrl || is_blank)
+                                              | view::char_to<value_type_t<id_type>>,
+                                                std::back_inserter(id));
+                detail::consume(stream_view | view::take_line_or_throw);
+            }
+        }
+
+        // Jump to sequence
+        while (!(is_char<'O'>(*std::ranges::begin(stream_view)) || options.embl_genbank_complete_header))
+            detail::consume(stream_view | view::take_line_or_throw);
+
+        // Sequence
+        detail::consume(stream_view | view::take_line_or_throw);
+        auto constexpr is_end = is_char<'/'> ;
+        if constexpr (!detail::decays_to_ignore_v<seq_type>)
+        {
+            auto constexpr is_legal_alph = is_in_alphabet<seq_legal_alph_type>;
+            std::ranges::copy(stream_view | std::view::filter(!(is_space || is_digit))
+                                          | view::take_until_or_throw_and_consume(is_end) // consume //
```

Comment:
```suggestion
                                          | view::take_until_or_throw_and_consume(is_end) // consume "//"
```
I was confused first :grin: :nail_care: